src/MainView.vala

```diff
@@ -148,7 +148,7 @@ public class OnlineAccounts.MainView : Gtk.Grid {
         remove_button.clicked.connect (() => {
             var message_dialog = new Granite.MessageDialog (
                 _("Remove “%s” from this device").printf (e_source.display_name),
-                _("This account will be removed and will no longer appear in apps on this device."),
+                _("This account will be removed, and no longer appear in apps on this device."),
```

Comment:
I think the original is good English (maybe a little formal) and possible easier for translation.

Reply:
Original has a not just redundant, but double "will".

Reply:
I don't have strong feelings about removing "will" here, but adding the comma is definitely not correct grammar. This is a single clause, not two independent clauses.

Reply:
Just added it for pause. Will remove.

Reply:
```suggestion
                _("This account will be removed and no longer appear in “Apps” on this device."),
```